econml/dml.py

```diff
@@ -284,8 +381,7 @@ def coef_(self):
                          model_final=FinalWrapper(),
                          discrete_treatment=discrete_treatment,
                          n_splits=n_splits,
-                         random_state=random_state,
-                         inference=inference)
+                         random_state=random_state)
 
     @property
     def coef_(self):
```

Comment:
I think the coef property here should return a tensor more inline with the original spec, where:
coef_[i, j, k] 
is the coefficient in front of feature k, for the effect of treatment j on outcome i. 

Currently it will return sth like:
coef_[i, j]
where j is a cryptic thing that corresponds to one of the cross-product features between X and T, i.e. sth like
coef_[i, j*d+k]
might correspond to the same as coef_[i, j, k]

But the latter is very hard to parse by a user and is not inline with the very original spec. 

I believe we might want to have coef_ available in a higher level at the linearCateEstimator and define the above interface there and pass.

Reply:
Created #114 so that we fix this in a future PR.